src/app/common/transactions/bitcoin/use-generate-bitcoin-tx.ts

```diff
@@ -20,7 +20,7 @@ interface GenerateNativeSegwitTxValues {
 export function useGenerateSignedNativeSegwitTx() {
   const {
     address,
-    publicKeychain: currentAddressIndexKeychain,
+    keychain: currentAddressIndexKeychain,
```

Comment:
I know you're just renaming this but there's a type warning in this file we could maybe fix while here? 

![Screenshot 2023-07-21 at 14 00 20](https://github.com/hirosystems/wallet/assets/2938440/0cb70d05-fd4b-4616-ade3-b151ddf4276a)